### PR TITLE
More accessibility changes

### PR DESCRIPTION
**Pull request by anabebs**
Added some error messages to forms to meet accessibility standards.

### CODE REVIEW

File: pages/people/[id]/records/index.tsx [hunk 1]
```diff
@@ -10,7 +8,6 @@ const AddNewRecordPage = (): React.ReactElement => {
   const personId = Number(query.id as string);
   return (
     <>
-      <Seo title={`Add a new record for #${query.id}`} />
```
Comment (sirLisko) [inline pages/people/[id]/records/index.tsx:13]: Don't we need a title for this page?

File: data/forms/warning-note-review.tsx [hunk 2]
```diff
@@ -59,7 +59,7 @@ const formSteps: FormStep[] = [
         name: 'nextReviewDate',
         label: 'Next review date',
         rules: {
-          required: true,
+          required: 'Pease add a valid date',
```
Comment (sirLisko) [inline data/forms/warning-note-review.tsx:62]: Sorry for being super picky, but some of the errors have a final `.` and some others don't 😅

File: pages/people/[id]/records/index.tsx (hunk 1 above)
Comment (anabebs) [reply pages/people/[id]/records/index.tsx:13]: i thought i'd reverted this change

File: data/forms/warning-note-review.tsx (hunk 2 above)
Comment (anabebs) [reply data/forms/warning-note-review.tsx:62]: ok will fix

**Review by sirLisko — APPROVED** [review]
👍

**Pull request merged**